README.md

```diff
@@ -6,22 +6,43 @@
 2. Describe why a virtual DOM might be useful
 3. Describe some of the tradeoffs of using a virtual DOM
```

Comment:
Do you achieve this objective in the document? Is the fact that it is a workaround, as you say at the end, the tradeoff?